server/src/bin/ingestion-microservice.rs

```diff
@@ -666,7 +670,7 @@ pub async fn readd_error_to_queue(
                 message.get_dataset_id(),
                 models::EventType::CardActionFailed {
                     chunk_id: message.get_chunkmetadata_id(),
-                    error: format!("Failed to upload chunk tracking_id {:?}: {:?}", id, error),
+                    error: format!("Failed to upload chunk {:?}: {:?}", id, error),
```

Comment:
Wait why remove this? There should always be a tracking_id here and its best to just say thats what the caused the issue no?